tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -87,3 +87,10 @@ Feature: An administrator wants to import a user using the commandline
     When a user is imported from path "simpleExport/unknown" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "user.json not found"
+
+  @issue-210
+  Scenario: importing with empty trash bin
```

Comment:
what user are trying to import can it be little bit clear?

Reply:
here an user is not trying to import something but User itself is being imported